drivers/mysql/mysql_test.go

```diff
@@ -196,25 +196,6 @@ func (suite *MysqlTestSuite) TestCreateSchemaTableIfNotExists() {
 	})
 }
 
-func (suite *MysqlTestSuite) TestLock() {
```

Comment:
Any way to retain the tests somehow or do they just become obsolete after this?

Reply:
I am thinking that they will just become obsolete. We plan to move away from advisory locks altogether. @isacikgoz and I were discussing that since this is an open tool, users might want to keep using the advisory lock implementation. But I think it will be too much overhead to maintain 2 separate locking implementations.

Ibrahim, let's discuss this when you are back. For now, only the test is removed. The lock code is still there.